File: SNS-Android App/app/src/main/java/com/codingstrings/sns/models/Notice.java
package com.codingstrings.sns.models;

public class Notice {
    public final String  _id ;
    public final String  message ;
    public final String  msubject ;
    public final String  seen;


    public Notice(String _id, String msubject, String message , String seen) {
        this._id = _id;
        this.message = message;
        this.msubject = msubject;
        this.seen = seen;
    }
}

File: Server/models/notice.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;


const NoticeSchema = new Schema({

  subject: {
    type: String,
    required: [true, 'Subject is Required']
  },
  msubject: {
    type: String,
    required: [true, 'Subject is Required']
  },
  year: {
    type: String,
    required: [true, 'Assignment Year is Required']
  },
  division:{
    type:String
  },
  batch:{
    type:String
  },
  message: {
    type: String
  },
  filepath: {
    type: [String]
  },
  created: {
    type: Date,
    default: Date.now
  }
});

const notice = mongoose.model('notice', NoticeSchema);
module.exports = notice;
